source/base/ReferenceCounter.ooc

```diff
@@ -22,6 +22,7 @@ ReferenceCounter: class {
 			}
 		}
 	}
+	count ::= this _count
```

Comment:
And again
```C
Numbers__Int props__Test___getvalue___impl(props__Test* this) {
    #line 3 "/home/seb/work/ooc-kean/props.ooc"
    return 2;
}

void props__Test___setvalue___impl(props__Test* this, Numbers__Int x) {
    #line 5 "/home/seb/work/ooc-kean/props.ooc"
    if (this->value == 2) {
...
```
but the `this->value` data member is never assigned any value.
Looks like it is not safe to use getters within setters ? Can you reproduce ?